### PR TITLE
update html and js files to set data appropriately

### PR DESCRIPTION
**Pull request by ToriForH**
#7638 

- [x] social network images could be updated
- [x] social network images could be deleted
- [x] when delete social network image it deleted from azure

### CODE REVIEW

**Review by holotsvan — APPROVED** [review]

File: service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java [hunk 1]
```diff
@@ -114,19 +116,50 @@ void getDefaultSocialNetworkImage() {
...
+        SocialNetworkImage image2 = SocialNetworkImage.builder()
+            .id(2L)
+            .hostPath("hostPath2")
+            .imagePath("imagePath2")
+            .build();
+        SocialNetworkImage image3 = SocialNetworkImage.builder()
+            .id(3L)
+            .hostPath("hostPath3")
+            .imagePath("imagePath3")
+            .build();
```
Comment (ABbondar) [inline service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java]: please, move it to model utils, we can create the list of images in model utils

File: service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java [hunk 2]
```diff
@@ -114,19 +116,50 @@ void getDefaultSocialNetworkImage() {
...
 
         verify(socialNetworkImageRepo).deleteById(idToDelete);
+        verify(fileService).delete(image.getImagePath());
+    }
+
+    @Test
+    void testDeleteImageNotFound() {
+        Long idToDelete = 1L;
+        when(socialNetworkImageRepo.findById(idToDelete)).thenReturn(Optional.empty());
+        assertThrows(NotFoundException.class, () -> socialNetworkImageService.delete(idToDelete));
```
Comment (ABbondar) [inline service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java:135]: please add verify

**Review by K4sik — APPROVED** [review]

File: service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java [hunk 3]
```diff
@@ -114,19 +116,50 @@ void getDefaultSocialNetworkImage() {
     @Test
     void testDelete() {
         Long idToDelete = 1L;
+        SocialNetworkImage image = ModelUtils.getSocialNetworkImage();
```
Comment (ABbondar) [inline service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java]: we can use static import here

File: service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java [hunk 4]
```diff
@@ -114,19 +116,50 @@ void getDefaultSocialNetworkImage() {
...
+            .hostPath("hostPath2")
+            .imagePath("imagePath2")
+            .build();
+        SocialNetworkImage image3 = SocialNetworkImage.builder()
+            .id(3L)
+            .hostPath("hostPath3")
+            .imagePath("imagePath3")
+            .build();
+        List<SocialNetworkImage> images = List.of(image1, image2, image3);
+        when(socialNetworkImageRepo.findAllById(listIds)).thenReturn(images);
```
Comment (ABbondar) [inline service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java:146]: please, add verify

Comment (ToriForH) [reply service/src/test/java/greencity/service/SocialNetworkImageServiceImplTest.java:146]: Updated due to requests

**Review by ABbondar — APPROVED** [review]

**Pull request merged**